bot/exts/evergreen/rps.py

```diff
@@ -0,0 +1,58 @@
+from random import choice
+
+from discord.ext import commands
+
+from bot.bot import Bot
+
+CHOICES = ['rock', 'paper', 'scissor']
+SHORT_CHOICES = ['r', 'p', 's']
+
+# Using a dictionary instead of conditions to check for the winner.
+WINNER_DICT = {
+    'r': {
+        'r': 0,
+        'p': -1,
+        's': 1,
+    },
+    'p': {
+        'r': 1,
+        'p': 0,
+        's': -1,
+    },
+    's': {
+        'r': -1,
+        'p': 1,
+        's': 0,
+    }
+}
+
+
+class RPS(commands.Cog):
+    """Rock Paper Scissor. The Classic Game!"""
+
+    @commands.command(case_insensitive=True)
+    async def rps(self, ctx: commands.Context, move: str) -> None:
+        """Play the classic game of Rock Paper Scissor with your own sir-lancebot!"""
+        move = move.lower()
+        player_mention = ctx.author.mention
+
+        if move not in CHOICES and move not in SHORT_CHOICES:
+            await ctx.send(f"Invalid move. Please make move from options: {', '.join(CHOICES).upper()}.")
+            return
```

Comment:
This should come out as an error
```suggestion
            raise commands.BadArgument(f"Invalid move. Please make move from options: {', '.join(CHOICES).upper()}.")
```